WELD.md

```diff
@@ -139,3 +104,36 @@ python weld.py
 ### Goal:
 
 1.  Track the submodule (recipe) version with the current data repository.
+
+## Step 5: Commit most weld results to Github
+
+### Procedure:
+Depending on the size of your dataset, it may not be practical to commit all weld results to Github. Our team has decided to commit all summary figures and results but not the per-site paint or spots data to Github. You can tailor the following commit code to suit your data storage preferences.
+
+```bash
+# EDIT {BATCH} IN FOLLOWING CODE TO MATCH YOUR DATASET
+# Add, commit, and push the weld results
+git add data
+git add figures
+git add results
+git reset data/0.site-qc/{BATCH}/paint/
+git reset data/0.site-qc/{BATCH}/spots/
```

Comment:
I am a bit weary of these instructions. I think that `git add data` will take a ton of time only to reset the folders that take the majority of the time.

Instead, consider adding only the relevant data files.

An alternative would be add those two folders to the `.gitignore` file. I think adding specific files is the better way, though.